core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/reset/AbstractClusterResetAction.java

```diff
@@ -0,0 +1,41 @@
+package com.sequenceiq.cloudbreak.core.flow2.cluster.reset;
```

Comment:
Full copy of AbstractClusterCreationAction, we should avoid copy-pastes if possible